project/profiles/capacity4more/behat/features/projects.feature

```diff
@@ -17,3 +17,20 @@ Feature: Testing the projects overview page.
     And   The window is maximized
     When  I visit the dashboard of project "Human Genome Project"
     Then  I should be able to toggle the project highlight link
+
+  @api
+  Scenario: Verify that PO can see draft projects, while regular user can not.
+    Given I am logged in as user "badhairday"
+    When  The project "Lusail City Hotel" status is changed by admin to "Draft"
+    Then  I visit "projects"
+    And   I should see "Lusail City Hotel"
+    And   I am logged in as user "isaacnewton"
+    And   I visit "projects"
+    And   I should not see "Lusail City Hotel"
+
+  @api
+  Scenario: Verify that regular user can see published projects.
+    Given I am logged in as user "badhairday"
+    When  The project "Lusail City Hotel" status is changed by admin to "Published"
```

Comment:
after this stage, the current user is the admin (user 1), so it has all permissions anyway ...